packages/emotion-theming/src/theme-provider.js

```diff
@@ -77,8 +77,18 @@ class ThemeProvider extends Component {
   render() {
     if (!this.props.children) {
       return null
+    } else if (Array.isArray(this.props.children)) {
```

Comment:
One trick I stole from preact-compat is to coerce it into an array.

Reply:
https://github.com/tkh44/holen/blob/master/src/index.js#L4

Reply:
Another option is to just not worry about it except for a warning. For React > 16 it wouldn't matter anyway, right?

Reply:
Yeah for 16+ it wouldn’t matter.

I went ahead and updated the impl to be simpler :) along the lines of your suggestion